pkg/event/controller.go

```diff
@@ -119,7 +126,7 @@ func (gen *controller) processNextWorkItem(ctx context.Context) bool {
 		return true
 	}
 	_, err := gen.eventsClient.Events(event.Namespace).Create(ctx, event, metav1.CreateOptions{})
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "not found") && !strings.Contains(err.Error(), "being terminated") {
```

Comment:
Can't we use `k8s.io/apimachinery/pkg/api/errors.IsNotFound()`?

Reply:
I updated it. Thank you!